hicommonwealth/commonwealth
# Fix app sidebar height

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/5723

## Description of Changes
- Fixes app level explore community and  create content sidebar heights based on if the user is inside/outside a community

## "How We Fixed It"

By some CSS and community level checks

## Test Plan
1. Login
2. Go to https://commonwealth.im/dashboard/for-you
3. Click on the plus button (create content) or explore button (explore communities) in the sidebar, the menu will open
4. Verify the sidebar menu is not cut off from bottom
5. Now go to any community page
6. Open the sidebar again and verify its not cut off from the bottom


## Deployment Plan
N/A

## Other Considerations
N/A
